_meetings/2020-06_ELIXIRAllHands2020.md

```diff
@@ -44,3 +45,19 @@ Bioschemas is an approach to make life sciences resources more discoverable by e
 The Bioschemas community now has proposals for 17 types that extend the Schema.org vocabulary to enable it to be used by the life sciences community. These include types such as Taxon, BioSample, and Phenotype, that are directly relevant to the Biodiversity community. There are deployments of these at the National Museum of Natural History of Paris and the Botanical Collections at the Meise Botanic Garden, Belgium.
 
 To enable ELIXIR communities to exploit Bioschemas markup to populate community search portals, we have developed the B-MUSE directed crawler. B-MUSE will extract Bioschemas markup from a given set of pages or identifier pattern and make the structured content available for search. This approach has been demonstrated using the ELIXIR TeSS training portal, and is in active development in other ELIXIR communities, viz Rare Diseases, Plants, and Intrinsically Disordered Proteins.
+
+## eDGAR and PhenPath resources join the Bioschemas Community ([poster](https://f1000research.com/posters/9-569))
```

Comment:
As the heading will be used an an internal link, maybe better to move the poster link to the corresponding text

Reply:
Fixed the link with the poster in the title